### PR TITLE
Implemented Python script to connect to QTM via LSL and log data

### LINKED ISSUE #9: Create a Basic Python Script to Connect to QTM
**Objective**: Write a basic Python script that uses pylsl to connect to the Lab Streaming Layer (LSL) from QTM and logs incoming data.
**Steps**

- [x] Use `pylsl `to resolve a stream of type 'Force'.
- [x] Create a basic loop to pull and log samples from the stream.
- [x] Add a print statement to show the incoming data.

### PR DESCRIPTION
**Pull request by Nissa9902**
Fixes #9 
## Summary

This pull request addresses issue #9 by implementing a Python script to connect to QTM via LSL and log incoming data.

### Changes Made

- Added `connect_to_qtm.py` script that utilizes `pylsl` to connect to the Lab Streaming Layer (LSL).
- The script resolves a 'Force' stream, pulls samples, and logs them with timestamps.

### Why This Change Is Needed

The new script is essential for logging and analyzing data streams from QTM, which is a key component of our project's data processing pipeline.

### Additional Notes

Please ensure that the LSL library is correctly installed and accessible to run this script successfully.

### CODE REVIEW

**Comment by Nissa9902** [comment]
fixes issue #9

File: backend/connect_to_qtm.py
Comment (JackMcPhillips1543) [inline backend/connect_to_qtm.py:1]: Remove some the comments, some of them are repetitive

**Comment by JackMcPhillips1543** [comment]
Also please add a README.md that explains how to download the proper requirements for Windows and Mac. For Mac the command is <code>brew install labstreaminglayer/tap/lsl</code>

**Comment by JackMcPhillips1543** [comment]
Also add .gitignore please

**Comment by git-voo** [comment]
Nice job @Nissa9902 !!
Just a few changes and we're good to go.

**About the Commit/Pull Request**
Your branch name and commit message look great. I like it.
1. Remember to always add me as a reviewer to your PRs. By doing this, I will get a notification that facilitates how promptly you get feedback.
2. Always assign the PR to yourself for the same reason as above. It helps us keep track, know who to reach out to for what, get notified promptly, and maintain professionalism.

**About the Code**
The code generally looks good. Seeing that we place so much value on clean codes and reusability, I'd ask you to group your streaming logic into a function.

See sample below:
`
def stream_data(stream_type='Force', sleep_time=0.1):
    """
    Connect to the specified data stream and print the data.

    :param stream_type: Type of the stream to connect to.
    :param sleep_time: Time in seconds to wait between pulling samples.
    """
`
 
You will see the importance of this modular approach better when we start streaming data to the frontend using websockets or APIs

**Review by git-voo — CHANGES_REQUESTED** [review]
Nice job @Nissa9902 !!
Just a few changes and we're good to go.

**About the Commit/Pull Request**
Your branch name and commit message look great. I like it.
1. Remember to always add me as a reviewer to your PRs. By doing this, I will get a notification that facilitates how promptly you get feedback.
2. Always assign the PR to yourself for the same reason as above. It helps us keep track, know who to reach out to for what, get notified promptly, and maintain professionalism.

**About the Code**
The code generally looks good. Seeing that we place so much value on clean codes and reusability, I'd ask you to group your streaming logic into a function.

See sample below:
````
def stream_data(stream_type='Force', sleep_time=0.1):
    """
    Connect to the specified data stream and print the data.

    :param stream_type: Type of the stream to connect to.
    :param sleep_time: Time in seconds to wait between pulling samples.
    """
``` 
 
You will see the importance of this modular approach better when we start streaming data to the frontend using websockets or APIs

File: backend/connect_to_qtm.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    try: 
+        while True: 
+            sample, timestamp = inlet.pull_sample()
+            print(f"Timestamp: {timestamp}, Sample: {sample}")
+            time.sleep(sleep_time)
+    except KeyboardInterrupt:
+       print("Streaming stopped by user.")
+    
+ if __name__ == "__main__":
+    stream_data()
```
Comment (git-voo) [inline backend/connect_to_qtm.py:29]: Just a note
Following our modular approach, in the future, stream_data() will be called from a separate component, a central component but I can understand why this is here for testing.

**Review by git-voo — APPROVED** [review]
Nice work!
Let's get this through.

**Review by git-voo — APPROVED** [review]

**Pull request merged**